tests/e2e-playwright/playwright.config.ts

```diff
@@ -0,0 +1,46 @@
+import { defineConfig, devices } from '@playwright/test'
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  // Look for test files in the "tests" directory, relative to this configuration file.
+  testDir: 'spec',
+
+  // Run all tests in parallel.
+  fullyParallel: true,
```

Comment:
we cannot run tests in parallel
```suggestion
  fullyParallel: false,
```